modules/ztp-image-based-upgrade-troubleshooting.adoc

```diff
@@ -7,6 +7,30 @@
 
 You can encounter issues during the image-based upgrade.
 
+[id="ztp-image-based-upgrade-troubleshooting-must-gather_{context}"]
+== Collect logs
+
+You can use the `oc adm must-gather` CLI to collect information for debugging and troubleshooting.
+
+.Procedure
+
+. Collect data specific to the {lcao} by running the following command:
++
+[source,terminal]
+----
+$ oc adm must-gather \
+  --dest-dir=must-gather/tmp \
+  --image=$(oc -n openshift-lifecycle-agent get deployment.apps/lifecycle-agent-controller-manager \
+  -o jsonpath='{.spec.template.spec.containers[?(@.name == "manager")].image}')
+----
+
+. Collect data about the OADP Operator by running the following command:
++
+[source,terminal]
+----
+$ oc adm must-gather --image=quay.io/konveyor/oadp-must-gather:latest
+----
```

Comment:
So we already collect some data related to oadp with the first command (basically that cmd will pull in everything IBU dev may need to troubleshoot)

The second  can be seen as "optionally collect even more oadp data". But maybe we are remove this altogether and just link Oadp's official page must-gather [page](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html-single/backup_and_restore/index#migration-using-must-gather_oadp-troubleshooting) instead?